### DISCUSSION TITLE
Enable/Incorporate Guidelines and Replay Templates

### DISCUSSION THREAD

**Discussion by SudharakaP** [opencollective/opencollective#4308] [Ideas]
Just got me thinking that we need to enable and incorporate more specific guidelines when interact with the community in general and how we should handle making issues as spam, closing issues etc. My thinking is that each persons perception of what is considered a issue that should be closed, that should be posted on stackoverflow instead, that should be a discussion, that is spam etc, can be highly subjective. When our team grows and more and more people interact on issues it can seem daunting and even rather difficult for us to act in consistent ways. 😄 

Therefore my suggestion is that we incorporate some maintainer reply templates; what I am talking about is something like these; https://github.com/jhipster/generator-jhipster/tree/main/.github/MAINTAINERS_REPLY_TEMPLATES. So if you are closing a issue or marking that as spam, or thinking that it doesn't belong in the repo etc etc; whatever the cause maybe, the template should be used as a comment before just closing it or making it as spam etc. This can easily be done using GitHub's save reply feature (https://docs.github.com/en/github/writing-on-github/working-with-saved-replies/using-saved-replies), so I don't think there's an overhead there. This would also enable the person who's replying to be always consistent on adhering to the guidelines put forth in the template. WDYT? 😄 😄 

If you think it's a good idea; I can volunteer to write up the reply templates. 😄 

@opencollective/core-developers 

@opencollective/core

**Comment by BenJam** [comment]
++ this is a great idea. I just rediscovered all my triage templates from GitHub and they used to save us a lot of time and build standards into our processes. 

Immediate first one for me is 

> please create issues in the main opencollective/opencollective' repo, we will transfer this there

**Reply by SudharakaP** [reply]
@BenJam : This is already done at https://github.com/opencollective/opencollective/pull/4309 so you have some issue templates at, https://github.com/opencollective/opencollective/tree/main/.github/MAINTAINER_REPLY_TEMPLATES. Now this doesn't include the "transfer this issue" template, but recently we also changed the GitHub issue format to v2 so when you go to a repo like front-end and try to create an issue you won't be able to but will see links like this; https://github.com/opencollective/opencollective-frontend/issues/new/choose. 😄
